.github/workflows/test-docker-container.yml

```diff
@@ -0,0 +1,38 @@
+name: Test docker image and container 
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  test-docker-image:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Test Image Build
+        id: image_build
+        working-directory: docker
+        run: |
+          IMAGE_TAG=test-image-${{ matrix.python-version }}
+          ./build_image.sh -py "${{ matrix.python-version }}" -t "${IMAGE_TAG}"
+          echo "IMAGE_TAG=${IMAGE_TAG}" >> $GITHUB_OUTPUT
+
+      - name: Container Healthcheck
+        working-directory: docker
+        run: ./test_container_health.sh ${{ steps.image_build.outputs.IMAGE_TAG }}
+
+      - name: Check Python Version in Container
+        working-directory: serve/docker
+        run: ./test_container_python_version.sh ${{ steps.image_build.outputs.IMAGE_TAG }} ${{ matrix.python-version }}
```

Comment:
@fabridamicelli Can you please change the working directory here. The run fails here

Reply:
yes! I just pushed the fix, so you can re-trigger the CI and it should (hopefully) work :)